src/IROperator.h

```diff
@@ -351,6 +351,11 @@ Expr requirement_failed_error(Expr condition, const std::vector<Expr> &args);
 
 Expr memoize_tag_helper(Expr result, const std::vector<Expr> &cache_key_values);
 
+/** Reset the counters used for random-number seeds in random_float/int/uint
+ * when no seed is explicitly passed in; this is used for multitarget compilation
```

Comment:
The counter is added even if a seed is passed in. The seed is additional.

Reply:
Done.